test/app.test.js

```diff
@@ -0,0 +1,9 @@
+/* global test,expect true */
```

Comment:
Just some quick things, if I remember correctly Jest by default looks for tests in the folders named __tests__. This can be handy for keeping tests in the hierarchy at the same level as the components that the tests relate to.

Also if we are being consistent with mobile, app.spec.js should be the test file name. Obviously that is just a code style which I'm pretty sure we haven't documented, so sorry about that!